app/src/main/java/com/teampophory/pophory/feature/album/cover/AlbumCoverEditActivity.kt

```diff
@@ -0,0 +1,188 @@
+package com.teampophory.pophory.feature.album.cover
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import androidx.lifecycle.flowWithLifecycle
+import androidx.lifecycle.lifecycleScope
+import androidx.viewpager2.widget.ViewPager2
+import com.teampophory.pophory.common.activity.hideLoading
+import com.teampophory.pophory.common.activity.showError
+import com.teampophory.pophory.common.activity.showLoading
+import com.teampophory.pophory.common.intent.intExtra
+import com.teampophory.pophory.common.intent.longExtra
+import com.teampophory.pophory.common.view.dp
+import com.teampophory.pophory.databinding.ActivityAlbumCoverEditBinding
+import com.teampophory.pophory.feature.album.cover.adapter.AlbumCoverEditAdapter
+import com.teampophory.pophory.feature.album.cover.model.AlbumCoverItem
+import com.teampophory.pophory.util.HorizontalMarginItemDecoration
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+@AndroidEntryPoint
+class AlbumCoverEditActivity : AppCompatActivity() {
+
+    private val binding by lazy {
+        ActivityAlbumCoverEditBinding.inflate(layoutInflater)
+    }
+    private val viewModel by viewModels<AlbumCoverEditViewModel>()
+    private val currentAlbumCoverId by intExtra(1)
+    private val albumId by longExtra(0)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        initObserve()
+    }
+
+    private fun initObserve() {
+        viewModel.albumEditState.flowWithLifecycle(lifecycle).onEach {
+            when (it) {
+                is AlbumEditState.Uninitialized -> {
+                    initViews()
+                }
+
+                is AlbumEditState.Loading -> {
+                    showLoading()
+                }
+
+                is AlbumEditState.Success -> {
+                    hideLoading()
+                    setResult(RESULT_OK)
+                    finish()
+                }
+
+                is AlbumEditState.Error -> {
+                    hideLoading()
+                    showError()
+                }
+            }
+        }.launchIn(lifecycleScope)
+    }
+
+    private fun initViews() {
+        initEditButton()
+        initAlbumCoverViewPager()
+        initSelectorClickListener()
+        setAlbumCoverData()
+    }
+
+    private fun initEditButton() {
+        binding.tvEditButton.setOnClickListener {
+            viewModel.patchAlbumCover(albumId, )
```

Comment:
```suggestion
            viewModel.patchAlbumCover(albumId)
```

뒤에 뭐 더 들어가야할 param있어?

Reply:
내 마음..
삭제하겠읍니다